client/tinychain/ml/linalg.py

```diff
@@ -206,16 +216,58 @@ def cond(p: Tensor, l: Tensor, u: Tensor, i: UInt):
         l=identity(x.shape[0], F32).as_dense().copy(),
         u=x.copy(),
         i=UInt(0),
+        num_permutations=UInt(0),
```

Comment:
Just FYI you can just write `0` here and not `UInt(0)` because the `UInt` annotation on the `step` op will ensure that the function receives a `UInt` when it's called at compile-time